modules/nf-core/paraphase/tests/main.nf.test

```diff
@@ -48,20 +50,21 @@ nextflow_process {
                 { assert snapshot(
                     file(process.out.json.get(0).get(1)).readLines()[0..39],
                     process.out.versions,
-                    file(process.out.vcf.get(0).get(1)).readLines()[0..962],
-                    process.out.bam ==~ "test.paraphase.bam",
-                    process.out.bai ==~ "test.paraphase.bam.bai",
+                    path(process.out.vcf.get(0).get(1)).linesGzip[0..962],
```

Comment:
This is a bit excessive, just a selection would be fine. Or use the vcf plugin for nf-test

Reply:
Can't use the nft-vcf because the variants are unstable after line 962, and can't use getVariants(size) because the VCF is malformed.

But I'll just snapshot the first variant.